### PR TITLE
Document the required command for c2c build in MacOS

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Document the required command for c2c build in MacOS
- Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/9431
- Resolves https://github.com/ballerina-platform/module-ballerina-c2c/issues/813

### CODE REVIEW

**Review by xlight05 — APPROVED** [review]

File: examples/docker-hello-world/docker_hello_world.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ For all the supported key value properties, see [Code to Cloud specification](ht
 ::: code Cloud.toml :::
 
 Execute the `bal build` command to build the Ballerina package. Code to Cloud generates only one container per package.
+>**Note:** For Mac users, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't yet support the M1 chip.
```
Comment (NipunaRanasinghe) [inline examples/docker-hello-world/docker_hello_world.md:14]: Should this be
```suggestion
>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't yet support the M1 chip.
```
? 

 I think this is not related to Mac users having intel chips, right?

**Review by tharindu-nw — APPROVED** [review]
